metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/DepublishPlugin.java

```diff
@@ -0,0 +1,50 @@
+package eu.europeana.metis.core.workflow.plugins;
+
+import eu.europeana.cloud.service.dps.DpsTask;
+import java.util.List;
```

Comment:
Unnecessary import.

Reply:
Weird my ide didn't handle it automatically